### ISSUE TITLE
spotlight: copy-pastable code output of queue for preparing announcements

### ISSUE THREAD

**Issue by Laogeodritt** [Worldbuilding/KazTron#99]
Per Yazuki request, `.spotlight queue copylist` or something like that, copy-pastable code output of the current queue in the form:

```
[item]. <@[user ID]> with **[Project Name]** (April 1 to April 3)
```

in order to ease preparation of announcement messages. (Automatic announcement is not preferred due to the need to edit and maintain the list as people change assignments/reject spotlights, etc.)

- possibly also filter by month? in case e.g. we still have 1-2 more spotlights this month but Yazuki only wants the queue for the next month

**Comment by Laogeodritt** [comment]
Todo: nicer date format for these

**Comment by Laogeodritt** [comment]
fixed in 951d4e1
